frontend/lib/src/components/elements/Toast/Toast.tsx

```diff
@@ -54,11 +54,15 @@ function generateToastOverrides(
       style: {
         width: "288px",
         marginTop: "8px",
-        borderRadius: "4px",
+        borderRadius: theme.radii.lg,
```

Comment:
Should we break down this into the four parts?

See https://github.com/streamlit/streamlit/blob/develop/frontend/lib/src/components/shared/Tooltip/Tooltip.tsx#L76

Reply:
Yeah it still generates warnings - switching to long specification